editor/src/clj/editor/resource.clj

```diff
@@ -441,10 +515,23 @@
                  (map (juxt #(str "/" (:path %)) :crc))
                  entries)})))
 
+(defn save-tracked?
+  "Checks if the specified Resource should be connected to the save system."
+  [resource]
+  (and (file-resource? resource)
+       (editable? resource)
+       (loaded? resource)))
+
```

Comment:
This logic was duplicated across places where `:auto-connect-save-data?` is `true` and where it is not.